docs/local-dev/cli-build-system.md

```diff
@@ -592,19 +592,26 @@ A complete launch configuration for VS Code debugging may look like this:
 ```json
 {
   "type": "node",
-  "name": "vscode-jest-tests",
+  "name": "vscode-jest-tests.v2",
   "request": "launch",
+  "env": {
+    // Needed to disable watch by default mode.
```

Comment:
Note that this comment makes this `json` has to be manually removed if a user just copies-and-pastes this block, since JSON doesn't support comments

Reply:
`launch.json` is a weird file, ends in .json but supports comments...